src/cudamatrix/cu-matrix.cc

```diff
@@ -196,6 +197,16 @@ template<class Real>
 template<class OtherReal>
 void CuMatrixBase<Real>::CopyFromMat(const CuMatrixBase<OtherReal> &M,
                                      MatrixTransposeType Trans) {
```

Comment:
Will fix the Trans, but regarding the M-- it's a little unfortunate, but
fairly large parts of the Kaldi code that use matrices and vectors, use
upper-case single-letter variables for matrices.  This is a holdover from
the pre-Google-style-guide days.  It would be a big change to convert all
this code to be fully compliant, so for now I think should continue to
allow (but discourage) upper-case single-letter variables that refer to
matrices.
Dan


On Wed, Aug 19, 2015 at 5:45 PM, Kirill Katsnelson <notifications@github.com
> wrote:

> In src/cudamatrix/cu-matrix.cc
> <https://github.com/kaldi-asr/kaldi/pull/83#discussion_r37485988>:
>
> > @@ -196,6 +197,16 @@ template<class Real>
> >  template<class OtherReal>
> >  void CuMatrixBase<Real>::CopyFromMat(const CuMatrixBase<OtherReal> &M,
> >                                       MatrixTransposeType Trans) {
>
> Lowercase m and trans.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/83/files#r37485988>.
>